proto/cmp/services/book/v1alpha/validate.proto

```diff
@@ -0,0 +1,66 @@
+syntax = "proto3";
+
+package cmp.services.book.v1alpha;
+
+import "cmp/types/v1alpha/common.proto";
+import "cmp/types/v1alpha/price.proto";
+
+message ValidationRequest {
+  // Message header
+  cmp.types.v1alpha.Header header = 1;
+
+  // Search ID that is returned in the search response message in the `metadata``
+  // (`SearchResponseMetadata`) field.
+  string search_id = 2;
+
+  // Result ID that is that is returned by `result_id` field of the search result
+  // messages like `AccommodationSearchResult`.
+  //
+  // Because search requests may contain multiple queries, for example one room
+  // (unit) for 2 adults and one room for 2 adults and 1 kid and another room for 1
+  // adult, here we are using a list of results IDs that the buyer wants to validate.
+  repeated int32 result_ids = 3;
+}
```

Comment:
if we have this example: for example one room (unit) for 2 adults and one room for 2 adults and 1 kid and another room for 1 adult, then we expect the same result_id with 3 different query_ids, to group the rooms to one bookable option. So I think we do not need repeated result_ids

Reply:
We should have a talk about multi-room in general. If only the best result in one hotel is being returned you might be right about merging this into one single offer, but when it comes down to having multiple combination choices it'll explode dramatically with the combination logic.